reportmanager/build/.dependencies

```diff
@@ -0,0 +1,204 @@
+dependencies:
```

Comment:
_.dependencies_ does not seems to required. Please added it into _.gitignore_ file